common/src/main/java/io/novafoundation/nova/common/utils/ListExt.kt

```diff
@@ -50,3 +50,7 @@ fun <T> List<T>.rangeCheck(fromIndex: Int, toIndex: Int) {
         toIndex > size -> throw IndexOutOfBoundsException("toIndex ($toIndex) is greater than size ($size).")
     }
 }
+
+fun <T> List<T>.countBy(predicate: (T) -> Boolean): Int {
+    return filter { predicate(it) }.size
+}
```

Comment:
Irrc there was `count {}` extension in stdlib already?

Reply:
Yes We has this extension, thank you. I'll change it